docs/index.md

```diff
@@ -1,71 +1,126 @@
+
+### Flexibility
+In order that as many types of interests as possible might be represented by the data model, we have maintained a high level of abstraction. In practice, that means identifying the components, common to all interests, which draw most attention from data users. These, we propose, are **dates, people and organisations**. The data model allows, though, for more detail to be added: for publishers to move to a lower level of abstraction. Data profiles can be created (external to datasets) which define the specific roles of dates, people and organisations in relation to different types of interest. For example, a recommended data profile for interests in the category 'employment and earnings' might be as follows.
+
+#### Data profile for 'Employment and earnings' interest
 
 ```eval_rst
 .. list-table::
     :header-rows: 1
-    :widths: 1 3 1 1
-
-    * - A
-      - B
-      - C
-      - D
-    * - Lorem
-      - Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec dictum urna non purus tristique pharetra.
-      - Yes
-      - False
-    * - Ipsum
-      - Mauris et lobortis nibh. Nullam vitae velit turpis. Vestibulum facilisis sit amet sapien nec maximus.
-      - No
-      - True
-    * - Dolor
-      - Pellentesque sit amet sapien tincidunt, fringilla dui id, porttitor purus. Nam tincidunt ac ex id porttitor. Praesent varius lectus nisl, ac luctus erat lacinia vitae.
-      - Yes
-      - True
+    :widths: 1 1 1 3
+
+    * - Object
+      - Property
+      - Value
+      - Description
+    * - OrganisationField
+      - fieldName
+      - Organisation benefitting
+      - The employer or organisation benefitting from the work.
+    * - OrganisationField
+      - fieldName
+      - Organisation paying
+      - The organisation paying for the work if different from that benefitting from it.
+    * - DateField
+      - fieldName
+      - Work completion date
+      - The date on which the work was completed. 
+    * - PersonField
+      - fieldName
+      - Person employed
+      - The person whose employment or earnings make this a registrable interest. 
 ```
 
-### Markdown reference links
+The values above would be published as part of a dataset, but their full meaning is provided by the 'description' in the data profile. Data profiles like this might be referenced from publishers' Publication Policies. 
 
-Link to [a ref](a-ref).
 
+### Open codelists
+To aid the flexibility and applicability of the model, we have opted for open codelists (codelists from which values can be chosen or which can be ignored in favour of custom values) where possible. For example an organisation, depending on the role it has in a particular interest, may be a 'Donor, Contracting organisation, Contracted organisation, Employer, Company, Organisation, Owner'. Or it may have another role entirely. 
 
-```eval_rst
-.. _a-ref:
-```
-#### Referenced section
 
-# Markdown inside an admonition
+## Outstanding issues
```

Comment:
Could we actually move all these to GitHub Issues, and just link to the issue queue here? I'm wary that sites like this can get stale.

Reply:
Hmmm. I see your point, but I think - for the moment at least - keeping them up front like this underlines the draft-iness of the data model. If we were to do another round of work on this, then these would be the top priority issues and having them represented as such in the rep would be a must. For the moment, having all the work and thinking on this summed up in a page and a spreadsheet, and no more, seems proportionate.

(I prob can be argued out of this position, if you're set on the alternative. But I thought I'd make my case!)